src/main/java/com/example/Employee.java

```diff
@@ -0,0 +1,63 @@
+package com.example;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+public class Employee {
+    private String employeeId;
```

Comment:
This Id should be a Long instead of string

Reply:
The `Employee` class already specifies the object and the fields, I think the `emloyeeId` is redundant, just '`id`' would be enough